inspector/src/components/graph/canvasGraphService.ts

```diff
@@ -65,15 +67,22 @@ export class CanvasGraphService {
             right: this._width,
         };
 
+        // we will now rescale the maximum for the playhead.
+        // Currently the scale factor is a constant but when we add panning this may become formula based.
+        const scaleFactor = 0.8;

```

Comment:
Would be great to move at the top of the file with other constants ? and move it back in if it changes.

Reply:
Done!